vnext/Microsoft.ReactNative.ComponentTests/IJSValueReader_h.h

```diff
@@ -0,0 +1,321 @@
+/* Header file automatically generated from IJSValueReader.idl */
+/*
+ * File built with Microsoft(R) MIDLRT Compiler Engine Version 10.00.0226 
+ */
+
+#pragma warning( disable: 4049 )  /* more than 64k source lines */
+
```

Comment:
This file is auto generated, add it to gitignore?

Reply:
This keeps happening on my dev-box. This was me. I'll file an issue. this file should be generated into the obj directory and should not live int he source tree. 
Danny's opinion: No build tool should write to the source tree :)

Reply:
Filed: #5332